rllib/policy/sample_batch.py

```diff
@@ -843,7 +845,8 @@ def _slice(self, slice_: slice):
                     for _ in range(l):
                         self._slice_map.append((i, sum_))
                     sum_ += l
-                self._slice_map.append((len(self[SampleBatch.SEQ_LENS]), sum_))
+                self._slice_map.append((len(self[SampleBatch.SEQ_LENS]) - 1,
```

Comment:
actually, I am not quite sure if I get this change here.
why are we leaving out the last seq?
can you help comment a bit?
thanks.

Reply:
Will do.